src/DssSpell.t.sol

```diff
@@ -552,13 +546,37 @@ contract DssSpellTest is DssSpellTestBase {
         address conduit     = addr.addr("RWA009_A_OUTPUT_CONDUIT");
         uint256 prevBalance = dai.balanceOf(conduit);
 
+        address rwaUrn009 = addr.addr("RWA009_A_URN");
+        (uint256 pink, uint256 part) = vat.urns("RWA009-A", address(rwaUrn009));
+
+        assertEq(prevBalance, 25_000_000 * WAD, "RWA009/bad-recipient-balance-before-spell");
+        assertEq(pink, 1 * WAD,                 "RWA009/bad-art-before-spell");
+        assertEq(part, 25_000_000 * WAD,        "RWA009/bad-ink-before-spell");
```

Comment:
Similar to Goerli tests we should not test the absolute values, but instead the difference. I think they've removed the 25m which is why this is failing actually.